src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportFlight.java

```diff
@@ -34,6 +34,16 @@ public SnapshotExportFlight(FlightMap inputParameters, Object applicationContext
     UUID snapshotId =
         UUID.fromString(inputParameters.get(JobMapKeys.SNAPSHOT_ID.getKeyName(), String.class));
 
+    boolean validatePrimaryKeyUniqueness =
+        Objects.requireNonNullElse(
+            inputParameters.get(
+                JobMapKeys.EXPORT_VALIDATE_PK_UNIQUENESS.getKeyName(), Boolean.class),
+            true);
+
+    if (validatePrimaryKeyUniqueness) {
+      addStep(new SnapshotExportValidateStep(bigQueryPdao, snapshotService, snapshotId));
```

Comment:
Do we foresee there being more validations?  We may want the name to be more specific (renaming steps is tough due to serialization issues so it may be worth being as explicit as possible up front)